controller/attribute/util.py

```diff
@@ -163,7 +164,7 @@ def extend_logs(
     if not attribute.logs:
         attribute.logs = logs
     else:
-        all_logs = [l for l in attribute.logs]
+        all_logs = [ll for ll in attribute.logs]
```

Comment:
is this intentionally renamed?

Reply:
yes, otherwise some extentions says it's not descriptive enough (red file color)